### DISCUSSION TITLE
Terminal watch face has white background

### DISCUSSION THREAD

**Discussion by russell-phillips** [InfiniTimeOrg/InfiniTime#2176] [General]
My son and I both have PineTime watches running InfiniTime. He's using the Terminal watch face, and it has a white background.

He'd like to change it back to a black background, but he doesn't know how he managed to get the white background. How does he change it back?

**Comment by NitroNils** [comment]
I suggest rebooting the device (by holding the button until it reboots). It sounds like the issue where colors are "reversed" if the rest of the watch interface is also like that.

**Comment by russell-phillips** [comment]
That worked, thank you :)
